packages/core/src/oidc/init.ts

```diff
@@ -344,5 +351,23 @@ export default function initOidc(
   });
   oidc.use(koaBodyEtag());
 
+  oidc.use(async (ctx, next) => {
+    await next();
+
+    // https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#registering-module-middlewares-helmet-ip-filters-rate-limiters-etc
```

Comment:
nit: Since we do have other middleware applied on oidc routes, no need to put the reference here？ Or maybe put it on top of all the middleware append statements? Also may be helpful to add some comments explaining why we need this piece of logic. Something like Record Daily active users for logging use.